.travis.yml

```diff
@@ -4,13 +4,11 @@ python:
   - "3.7"
 
 env:
-  - DOCKER_COMPOSE_VERSION=1.25.0 RUNS=10  SERVICE=<your first service>
-#  - DOCKER_COMPOSE_VERSION=1.25.0 RUNS=10  SERVICE=<your second service>
-  
-  - DOCKER_COMPOSE_VERSION=1.25.0 RUNS=20  SERVICE=all
+  - DOCKER_COMPOSE_VERSION=1.25.0 RUNS=10 SERVICE=<service name> VULNS=<vulns number>
```

Comment:
It was intentional to show how to add multiple services (with the commented line)